src/pages/groupComparison/pathwayMapper/GroupComparisonPathwayMapper.tsx

```diff
@@ -19,11 +19,21 @@ import { DefaultTooltip } from 'cbioportal-frontend-commons';
 import GroupComparisonStore from '../GroupComparisonStore';
 import { GeneOptionLabel } from 'pages/resultsView/enrichments/EnrichmentsUtil';
 import { SingleGeneQuery } from 'shared/lib/oql/oql-parser';
+import { PathwayMapParams } from '../GroupComparisonPage';
 
 interface IGroupComparisonPathwayMapperProps {
     activeGroups: ComparisonGroup[] | undefined;
     genomicData: AlterationEnrichmentRow[];
     store: GroupComparisonStore;
+    currentPathway: string;
+    rankingChoices: PathwayMapParams;
+    updateRankingChoices: (
+        drowDownTitle: string,
+        isAlterationEnabled: number,
+        considerOnlyTCGAPanPathways: boolean,
+        isPercentageMatch: number,
+        selectedPathway: string
+    ) => void;
```

Comment:
replace this with a single `userSelectionStore: GroupComparisonPathwayMapperUserSelectionStore` prop